### PR TITLE
Add Cousine Powerline

### PR DESCRIPTION
**Pull request by revolutionnaire**
I've also created a new specimen sample page using [Middleman](https://www.middlemanapp.com).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,37 @@
...
-    Powerline](https://github.com/powerline/fonts/tree/master/LiberationMono)
-  * `PT Mono` - PT Mono for Powerline
-  * `Source Code Pro` - [Source Code for Powerline](https://github.com/powerline/fonts/tree/master/SourceCodePro)
-  * `Ubuntu Mono` - [Ubuntu Mono derivative
-    Powerline](https://github.com/powerline/fonts/tree/master/UbuntuMono)
-
-See [preview.html](https://rawgit.com/wernight/powerline-web-fonts/master/preview.html) and [Slant](http://www.slant.co/topics/67/~programming-fonts) if you don't know which to choose.
+There are several Powerline-enabled typefaces to choose from:
+
+  * **Anonymous Pro**  [Anonymice Powerline](https://github.com/powerline/fonts/tree/master/AnonymousPro)
```
Comment (wernight) [inline README.md]: I kept them as code because you need to copy/paste that specific string. Could be **`...`**.

File: README.md [hunk 2]
```diff
@@ -1,42 +1,37 @@
...
 
 ## Usage example
 
 ### Usage example for [Secure Shell](https://chrome.google.com/webstore/detail/secure-shell/pnhechapfaindjhompbnflcldabbghjo)
 
-  - Options:
-      - font-family: `"Source Code Pro", monospace`
-      - user-css: `https://cdn.rawgit.com/wernight/powerline-web-fonts/4c2fbd9a52a443fbdf00c9eb79e615f1bed3a55c/PowerlineFonts.css`
+ - Go to Options under Secure Shell or chrome-extension://pnhechapfaindjhompbnflcldabbghjo/html/nassh_preferences_editor.html
+ - Change `font-family` to `"Cousine Powerline", "Everson Mono", FreeMono, "Menlo", "Terminal", monospace`
```
Comment (wernight) [inline README.md]: I'd keep "Source Code Pro" as the default because it's more popular right now.

**Comment by revolutionnaire** [comment]
I just tested this on my version of vim and Cousine Powerline practically takes out a column or two from vim.

**Comment by wernight** [comment]
NP thanks for your work. I'll look into updating the readme a bit.

**Pull request closed without merging**